### PR TITLE
Ruby200 latest bundler

### PR DESCRIPTION
**Pull request by jrafanie**
Package up bundler 1.7.7, net-http-persistent 2.9.4, and thor 0.19.1 for use with the ruby200 scl.

Ran f19.rb from https://github.com/fedora-ruby/fermig to clean up some of the convention changes.

### CODE REVIEW

**Comment by jrafanie** [comment]
@brandondunne, please ack when you get to test the resulting rpms on our internal CI: https://copr.fedoraproject.org/coprs/jrafanie/ruby200-bundler/

cc @jprause @jvlcek

**Comment by bdunne** [comment]
@jrafanie Do you need to include the `rubygem-bundler/bundler-1.3.5.gem`?

**Comment by jrafanie** [comment]
Yeah, for history.  This [commit](https://github.com/jrafanie/srpms/commit/1b4df99741aebc10c88d9393c6c5376377b26e0d) adds bundler at 1.3.5 as a baseline and then I update it.  Otherwise it's hard to tell what I changed going from 1.3.5 to 1.7.7.  Granted, this isn't distgit, I wanted to make sure I included in the .gem files so it makes it easier to rebuild later on.

File: rubygem-bundler/rubygem-bundler.spec [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+for dependency in \\\
+  net-http-persistent \\\
+  thor \
+do \
+  for fileordir in \\\
+    %{gem_dir}/gems/$dependency-*/lib/* \
+  do \
+    ln -s -f $fileordir %{gem_libdir}/bundler/vendor/$(basename "$fileordir") \
+  done \
+done
```
Comment (bdunne) [inline rubygem-bundler/rubygem-bundler.spec:19]: ...wow

**Comment by bdunne** [comment]
@jrafanie everything looks good here, I'll comment once I get it installed on a system. :+1:

**Comment by bdunne** [comment]
Installed on another system.  Everything looks good to merge. :cake:

**Pull request merged**